REVEL.pm

```diff
@@ -77,6 +77,20 @@ sub new {
 
   $self->get_user_params();
 
+ # get headers
+  my $file = $self->params->[0];
+  open HEAD, "tabix -fh $file 1:1-1 2>&1 | ";
+  while(<HEAD>) {
+    next unless /^\#/;
+    chomp;
+    $_ =~ s/^\#//;
+    $self->{headers} = [split];
+  }
+  close HEAD;
+
+  die "ERROR: Could not read headers from $file\n" unless defined($self->{headers}) && scalar @{$self->{headers}};
+  $self->{revel_file_columns} = scalar @{$self->{headers}};
```

Comment:
Suggest checking if number of columns 7 or 8 here and a comment why 7 or 8 columns.